src/TMap.cpp

```diff
@@ -2563,7 +2564,8 @@ bool TMap::readXmlMapFile(QFile& file, QString* errMsg)
         // until now:
         isLocalImport = true;
         mpProgressDialog = new QProgressDialog(tr("Importing XML map file for use in %1...").arg(mProfileName), QString(), 0, 0);
-        mpProgressDialog->setWindowTitle(tr("Map import", "This is a title of a progress dialog."));
+        //: This is a title of a progress dialog.
```

Comment:
Actually, given the other two texts it would probably be better to revise the comment:
```suggestion
        //: This is a title of a progress window.
```